### ISSUE TITLE
Installation bug

### ISSUE THREAD

**Issue by huard** [Ouranosinc/Magpie#27]
python setup.py install fails at line
```
from magpie.__meta__ import __version__, __author__, __email__
```
because magpie and its dependencies are not yet installed when first running it.

```
Traceback (most recent call last):
  File "setup.py", line 10, in <module>
    from magpie.__meta__ import __version__, __author__, __email__
  File "/home/david/src/Magpie/magpie/__init__.py", line 5, in <module>
    from api_except import *
  File "/home/david/src/Magpie/magpie/api_except.py", line 1, in <module>
    from pyramid.httpexceptions import *
ImportError: No module named pyramid.httpexceptions
```

**Comment by Zeitsperre** [comment]
If installing Magpie from Docker, the newest image available is based on "push-phoenix-btn", while the next supported image is "0.4.0" (23 March 2018). The current "latest" image is based on "Master" from around 6 months ago.

Should I trigger a new "latest" image based on Master?

**Comment by Zeitsperre** [comment]
For reference: https://hub.docker.com/r/pavics/magpie/builds/

**Comment by fmigneault** [comment]
"push-phoenix-btn" is under development for testing (not yet ready).
I would stick to 0.4.0 until this feature is complete

**Comment by fmigneault** [comment]
resolved by PR #33

**Comment by fmigneault** [comment]
resolved by PR #33
